### ISSUE TITLE
Inconsistency: the 'Upload image from computer' button is the only one that displays its tooltip below the button

### ISSUE THREAD

**Issue by marcelmroz** [ckeditor/ckeditor5#16131]
## 📝 Provide detailed reproduction steps (if any)

1. Open [Editor with a bottom toolbar and button grouping](https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/bottom-toolbar-editor.html)
2. Hover over the Upload image from computer dropdown

### ✔️ Expected result

The tooltip should be displayed in the same way as it is for the other buttons.

### ❌ Actual result

https://github.com/ckeditor/ckeditor5/assets/103954333/4f2e42fe-b5f2-4d9e-a18a-1e38e0fc4a0c

## 📃 Other details

Browser: Any
Not a recent regression - the tooltip is also missing in older docs versions

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
